cypress/realm-export.json

```diff
@@ -0,0 +1,2104 @@
+{
```

Comment:
I am curious how this was generated! But all looks good to me

Reply:
I exported from the keycloak portal.